.github/workflows/test.yml

```
@@ -109,21 +109,38 @@ jobs:
     - name: Install autoreject
       run: |
         pip install -e .[test]
+    - name: Clone MNE-Python
+      run: git clone --depth 1 --single-branch --branch main https://github.com/mne-tools/mne-python.git -b main
```

why specify the branch twice?

```suggestion
      run: git clone --depth 1 --single-branch --branch main https://github.com/mne-tools/mne-python.git
```